zingo_common_components/src/protocol.rs

```diff
@@ -0,0 +1 @@
+pub mod activation_heights;
```

Comment:
What are the tradeoffs between putting the gating attribute inside and outside the file?